packages/story-editor/src/components/canvas/mediaRecordingLayer.js

```diff
@@ -202,26 +205,40 @@ function MediaRecordingLayer() {
             <Wrapper>
               {!isImageCapture && (
                 <>
-                  {mediaBlobUrl && (
+                  {mediaBlobUrl && hasVideo && (
                     <VideoMode value={!isGif} onChange={onToggleVideoMode} />
                   )}
                   <VideoWrapper>
-                    {mediaBlobUrl && (
-                      <>
-                        {/* eslint-disable-next-line jsx-a11y/media-has-caption -- We don't have tracks for this. */}
-                        <Video
-                          ref={videoRef}
-                          src={mediaBlobUrl}
-                          muted={isMuted}
-                          loop={isGif}
-                          tabIndex={0}
-                        />
-                        {!isGif && <PlayPauseButton videoRef={videoRef} />}
-                      </>
-                    )}
-                    {!mediaBlob && !mediaBlobUrl && liveStream && (
-                      <Video ref={streamRef} muted />
-                    )}
+                    {mediaBlobUrl &&
+                      (hasVideo ? (
+                        <>
+                          {/* eslint-disable-next-line jsx-a11y/media-has-caption -- We don't have tracks for this. */}
+                          <Video
+                            ref={videoRef}
+                            src={mediaBlobUrl}
+                            muted={isMuted}
+                            loop={isGif}
+                            tabIndex={0}
+                          />
+                          {!isGif && <PlayPauseButton videoRef={videoRef} />}
+                        </>
+                      ) : (
+                        <>
+                          {/* eslint-disable-next-line jsx-a11y/media-has-caption -- No captions wanted/needed here. */}
+                          <audio controls="controls" src={mediaBlobUrl} />
```

Comment:
See this comment https://github.com/GoogleForCreators/web-stories-wp/pull/11677#issuecomment-1156455571. 

We do not show the built in players for video / audio, as they look very different in different browsers. I would recommending show the visualizer and reuse play / pause icon that is used in video.

Reply:
Note: visualizer will display just the slice of the audio, not the whole timeline with scrubbing abilities - that would be actually useful but I think we might introduce the audio trimmer in the near future? @swissspidy

IMO for a final file preview, the default audio player with a scrubber is far more useful than a slice visualizer without a scrubber. Only trimmer would be superior.

Reply:
We do not have scrubbing for video. So for consistency, let's not have it here. I think once trimming is introduced, then we can add scrubbing.

Reply:
I'm fine with having the default audio player here with a scrubber & time display, until we come up with something better in the future.

Reply:
Trimming for video is introduced as we speak.